examples/visualization/brain.py

```diff
@@ -0,0 +1,65 @@
+"""
+.. _ex-brain:
+
+===============================
+Plotting with ``mne.viz.Brain``
+===============================
+
+In this example, we'll show how to use :class:`mne.viz.Brain`.
+"""
+# Author: Alex Rockhill <aprockhill@mailbox.org>
+#
+# License: BSD-3-Clause
+
+# %%
+
+import os.path as op
+import matplotlib.pyplot as plt
+
+import mne
+from mne.datasets import sample
+
+print(__doc__)
+
+data_path = sample.data_path()
+sample_dir = op.join(data_path, 'MEG', 'sample')
+subjects_dir = op.join(data_path, 'subjects')
+evoked = mne.read_evokeds(op.join(sample_dir, 'sample_audvis-ave.fif'))[0]
+trans = mne.read_trans(op.join(sample_dir, 'sample_audvis_raw-trans.fif'))
+stc = mne.read_source_estimate(op.join(sample_dir, 'sample_audvis-meg'))
+
+brain = mne.viz.Brain('sample', subjects_dir=subjects_dir,
+                      alpha=0.5, cortex='low_contrast')
+
+# add sensors
+# brain.add_sensors(
+#     epochs.info, trans, colors=dict(eeg='green', grad='blue', mag='purple'),
+#     ch_types=dict(meg=True, eeg=True))
+
+# mark a region
+brain.add_label('BA3a', hemi='lh')
+
+# rotate brain by setting the camera position
+brain.show_view(azimuth=50, elevation=80, distance=250)
+
+# add a head
+# brain.add_head(color='red', alpha=0.1)
+
+# take a screenshot
+img = brain.screenshot()
+fig, ax = plt.subplots()
+ax.imshow(img)
+ax.axis('off')
+fig.suptitle('Brain')
+
+# %%
+# add an stc
+brain = mne.viz.Brain('sample', subjects_dir=subjects_dir)
+
+fmin, fmax = stc.data.min(), stc.data.max()
+for i, hemi in enumerate(('lh', 'rh')):
+    hemi_data = stc.data[:len(stc.vertices[i])]
+    hemi_vertices = stc.vertices[i]
+    brain.add_data(hemi_data, fmin=fmin, hemi=hemi, fmax=fmax,
+                   vertices=hemi_vertices,
+                   smoothing_steps='nearest', time=stc.times)
```

Comment:
```suggestion
                   smoothing_steps='nearest', time=stc.times)


# %%
# Highlight a region on the brain
# -------------------------------
#
# It can be useful to highlight a region of the brain to visually identify
# if the source activity is in an expected region.
# To highlight a region on the brain you can use the ``add_label`` method.
# Where do we get these names from?
# Is there an MNE tutorial listing these names in a table?
# How to I understand what BA3a is?
# Is there an external website that exaplains these acronyms?

# mark a region
brain.add_label('BA44', hemi='lh', color='green', borders=True)                   
```

Reply:
I think this might be more appropriate to run at the end, so we can demonstrate how the region of interest is overlaid on the source results.